app/templates/pdf/event_invoice.html

```diff
@@ -144,6 +144,7 @@
   </head>
   <body>
     <div id="logo">
+      <img src="{{admin_info.admin_billing_logo}}" alt="organization logo"  /><br>
```

Comment:
Only add if admin_billing_logo is non null

Reply:
ok